Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/MenuViewController.swift

```diff
@@ -1,405 +1,287 @@
 
-private class MenuCell: UITableViewCell {
-    let iconView = UIImageView().then {
-        $0.snp.makeConstraints { make in
-            make.width.equalTo(50)
+struct MenuItemHeaderView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+    var icon: UIImage
+    var title: String
+    
+    var body: some View {
+        HStack(spacing: 14) {
+            RoundedRectangle(cornerRadius: 8, style: .continuous)
+                .fill(Color(Theme.of(nil).colors.addressBar))
+                .frame(width: 32, height: 32)
+                .overlay(
+                    Image(uiImage: icon)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .padding(6)
+                )
+                .shadow(color: Color.black.opacity(0.1), radius: 1, x: 0, y: 1)
+            Text(verbatim: title)
         }
-        $0.contentMode = .center
+        .foregroundColor(Color(Theme.of(nil).colors.tints.home))
     }
-    
-    let labelView = UILabel()
-    let iconLength: CGFloat = 50.0
-    
-    let toggleButton = UISwitch().then {
-        $0.isHidden = true
-        $0.transform = CGAffineTransform(scaleX: 0.8, y: 0.8)
-        // At the moment, this toggle is only for visuals, tapping on the cell itself is not different
-        // than tapping on the switch.
-        $0.isUserInteractionEnabled = false
-    }
-    
-    private let spinner = UIActivityIndicatorView().then {
-        $0.snp.makeConstraints { make in
-            make.size.equalTo(24)
+}
+
+private struct MenuView<Content: View>: View {
+    var content: Content
+    @ObservedObject var themeNormalMode = Preferences.General.themeNormalMode
+    var body: some View {
+        ScrollView(.vertical) {
+            content
+                .padding(.vertical, 8)
+                .frame(maxWidth: .infinity)
         }
-        $0.hidesWhenStopped = true
-        $0.isHidden = true
-    }
-    
-    let stackView = UIStackView().then {
-        $0.spacing = 4
-        $0.alignment = .center
     }
-    
-    var isLoading = false {
-        didSet {
-            labelView.isEnabled = !isLoading
-            toggleButton.isHidden = isLoading
-            spinner.isHidden = !isLoading
-            
-            // on iOS12 if `isHidden` property is ignored if the spinner is animating.
-            if isLoading {
-                spinner.startAnimating()
-            } else {
-                spinner.stopAnimating()
-            }
+}
+
+struct MenuItemButton: View {
+    var icon: UIImage
+    var title: String
+    var action: () -> Void
+    
+    var body: some View {
+        Button(action: action) {
+            MenuItemHeaderView(icon: icon, title: title)
+                .padding(.horizontal, 14)
+                .frame(maxWidth: .infinity, minHeight: 48.0, alignment: .leading)
         }
+        .buttonStyle(TableCellButtonStyle())
+        .accentColor(Color(Theme.of(nil).colors.tints.home))
```

Comment:
why do we use `.accentColor` here instead of `.background(`

Reply:
This `accentColor` was to set colors of Toggle's to brave's orange, but it doesn't seem to work as Michal tested.

`background` places a `View` behind this view, which isn't what we wanted in this case